README.md

```diff
@@ -48,6 +50,9 @@ use it to create an another task for that instance.
 Note that the task name should always be unique in an instance, because Tinybench stores the tasks based
 on their names in a `Map`.
 
+Also note that `tinybench` does not log any result by default. You can extract the relevant stats
+from `bench.results` after running the benchmark, and process them however you want.
```

Comment:
`Task.results` and not bench.results.

Reply:
Well Task.result, but Bench.result__s__ if I read well the documentation below.

Reply:
Oh i thought you just wanted to process one task result! it's okay, you can go with bench.results if you want to process all results.